repo_manager/schemas/collaborator.py

```diff
@@ -0,0 +1,52 @@
+from typing import Self
+
+from github import Github
+
+from repo_manager.utils import get_inputs
+
+from pydantic import BaseModel  # pylint: disable=E0611
+from pydantic import Field, field_validator, model_validator
+
+
+class Collaborator(BaseModel):
+    type: str = Field("team", description="Type of reviewer, can be `user` or `team`")
+    name: str = Field(
+        "andrewthetechie",
```

Comment:
Please dont use my github username as the default here, I don't want to accidentally get added as a collaborator anywhere :D